api/BatteryManager.json

```
@@ -73,17 +37,16 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": "3.0",
-            "notes": "Values for <a href='https://developer.mozilla.org/docs/Web/API/BatteryManager/chargingTime'><code>BatteryManager.chargingTime</code></a> and <a href='https://developer.mozilla.org/docs/Web/API/BatteryManager/dischargingTime'><code>BatteryManager.dischargingTime</code></a> are always equal to <code>Infinity</code>."
+            "version_added": "3.0"
           },
           "webview_android": {
-            "version_added": "40"
+            "version_added": "38"
           }
         },
         "status": {
           "experimental": false,
```

I'm fine with undeprecating this, but it's single-implementation now, so let's flip this? Probably ought to be done for this and its descendants.

```suggestion
          "experimental": true,
```

---

I can't find where we had the discussion, but this is a perfect example of the case where I think calling single-implementation feature experimental seems inaccurate. There really is no experimenting going on, and web developers shouldn't expect the behavior to change. The spec probably won't change much either.

Web developers do need to be aware that the feature isn't supported for a large fraction of their users, but that's something else which we don't have a flag for in BCD, I think.

I'll set `experimental` to true if that's the current guidelines, but how should we go about changing that?

---

We never actually resolved https://github.com/mdn/browser-compat-data/issues/6905 with a real definition. It's more than just a guideline, because it also impacts the schema docs.

I think the interim approach has been, basically, experimental is a proxy for whether or not there's multiple implementations. I guess I don't care that much and can merge without it, but…

I do have an incomplete rough-draft of a guideline that captures more nuance like we discussed in #6905, but it's got some problems (I ought to open a PR to actually discuss that). To keep symmetry with the other removal guidelines, I've got something along the lines of it being OK to flip `experimental` to `false` for features which have not been introduced into a new browser in the last two years, which doesn't really help here anyway. Edge adds a wrinkle to a lot of stuff I hadn't really thought through when we last discussed the prospect of revising experimental (though I guess it's a wrinkle that would go away in January 2022).

---

@ddbeck I'd be happy to review that, however unsatisfactory you find it. If I were to state my position shortly, it would be that old and experimental are incompatible. If something first shipped over two years ago and there isn't much activity on the spec or implementation side, then it's not experimental.